Plugins/GitUIPluginInterfaces/ObjectId.cs

```diff
@@ -104,22 +104,14 @@ public static bool TryParse([CanBeNull] string s, int offset, out ObjectId objec
                 return false;
             }
 
-            var success = true;
-
             var i1 = HexToUInt32(offset);
             var i2 = HexToUInt32(offset + 8);
             var i3 = HexToUInt32(offset + 16);
             var i4 = HexToUInt32(offset + 24);
             var i5 = HexToUInt32(offset + 32);
 
-            if (success)
-            {
-                objectId = new ObjectId(i1, i2, i3, i4, i5);
-                return true;
-            }
-
-            objectId = default;
-            return false;
+            objectId = new ObjectId(i1, i2, i3, i4, i5);
+            return true;
```

Comment:
Some temp testing code that should have been removed (or check added @drewnoakes ?)